docs/experiments/categories/gcp/gcp-vm-instance-stop/auto-scaling.yaml

```diff
@@ -0,0 +1,30 @@
+## scale up and down to maintain the available instance counts
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: gcp-vm-instance-stop-sa
+  experiments:
+  - name: gcp-vm-instance-stop
+    spec:
+      components:
+        env:
+        # tells if instances are part of autoscaling group
+        # supports: enable, disable. default: enable
```

Comment:
The default value of `AUTO_SCALING_GROUP` is `disable`, can you please update the comment?